python/cloudtik/runtime/common/scripts/cloud-storage-fuse.sh

```diff
@@ -52,12 +61,37 @@ function configure_azure_blob_fs() {
         return
     fi
 
-    fuse_connection_cfg=${USER_HOME}/fuse_connection.cfg
-    echo "accountName ${AZURE_STORAGE_ACCOUNT}" > ${fuse_connection_cfg}
-    echo "authType MSI" >> ${fuse_connection_cfg}
-    echo "identityClientId ${AZURE_MANAGED_IDENTITY_CLIENT_ID}" >> ${fuse_connection_cfg}
-    echo "containerName ${AZURE_CONTAINER}" >> ${fuse_connection_cfg}
+    fuse_connection_cfg=${USER_HOME}/blobfuse2_config.yaml
+    cat>${fuse_connection_cfg}<<EOF
+allow-other: true
+logging:
+    type: syslog
+libfuse:
+    attribute-expiration-sec: 240
+    entry-expiration-sec: 240
+    negative-entry-expiration-sec: 120
+file_cache:
+    path: /mnt/ramdisk/blobfusetmp
+attr_cache:
+  timeout-sec: 7200
+azstorage:
+    type: ${BLOBFUSE_STORAGE_TYPE}
+    account-name: ${AZURE_STORAGE_ACCOUNT}
+    appid: ${AZURE_MANAGED_IDENTITY_CLIENT_ID}
+    endpoint: https://${AZURE_CONTAINER}.${AZURE_ENDPOINT}.core.windows.net
+    mode: msi
+    container: ${AZURE_CONTAINER}
+    update-md5: false
+    validate-md5: false
+    virtual-directory: true
+components:
+    - libfuse
+    - file_cache
+    - attr_cache
+    - azstorage
+EOF
```

Comment:
How about the case for when using configuring account key (secret) instead of using our default MSI?